homeassistant/components/keba/__init__.py

```diff
@@ -50,6 +50,7 @@
     "enable": "async_enable_ev",
     "disable": "async_disable_ev",
     "set_failsafe": "async_set_failsafe",
+    "set_text": "async_set_text",
```

Comment:
I think this line should be removed too. Sending a message is not done via a service call but via notify?

Reply:
Yes, thanks!  Fixed now.